### PR TITLE
Support CPU Pinning 

### PR DESCRIPTION
**Pull request by a110605**
### Summary

Support CPU Pinning 
- support enable / disable CPU manager in host page and see status in detail page
- support enable CPU Pinning in create VM page.
- support toggle CPU Pinning in edit VM page
- filter out un-migratable VM when do VM migration

Note. wait for backend PR merge : https://github.com/harvester/harvester/pull/6190

#### PR Checklist
- Is this a multi-tenancy feature/bug?
    - [ ] Yes, the relevant RBAC changes are at:
- Do we need to backport changes to the [old Rancher UI](https://github.com/rancher/u), such as RKE1?
    - [ ] Yes, the relevant PR is at:
- Are backend engineers aware of UI changes?
    - [x] Yes, the backend owner is: @brandboat 

Related Issue # https://github.com/harvester/harvester/issues/6056

### Screenshot/Video
1. Enable / disable CPU manager in host page
https://github.com/user-attachments/assets/39ac62ef-0956-4fab-ae66-f01b2430da40

2. Enable CPU pinning when VM create, disable it afterward
disable_enable_cpu_pinning

3. Hint user to turn on CPU manager if no node enabled CPU Manager

https://github.com/user-attachments/assets/2a204edc-1211-46a2-9111-4811bd653bd7

4. Filter out un-migratable node (harvster-node-0 is not enable CPU manager)

https://github.com/user-attachments/assets/5d097064-717a-4570-8f5a-33973219b51c

### CODE REVIEW

File: pkg/harvester/l10n/en-us.yaml [hunk 1]
```diff
@@ -495,6 +500,10 @@ harvester:
         ShutdownAndRestartAfterEnable: Shutdown and Restart After Enable
         ShutdownAndRestartAfterDisable: Shutdown and Restart After Disable
         Shutdown: Shutdown
+    cpuPinning:
+      label: Enable CPU Pinning
+      tooltip: Enable CPU Pinning brings better performance and reduce latency for the virtual machine
```
Comment (a110605) [inline pkg/harvester/l10n/en-us.yaml:505]: @brandboat , see if this tooltip is okay for you or have any suggestion ?

**Review by brandboat — APPROVED** [review]
LGTM, thanks !
backend pr has been merged, (master and v1.4 branch)

**Review by torchiaf — APPROVED** [review]
LGTM

**Comment by a110605** [comment]
@mergify backport release-harvester-v1.4

**Pull request merged**